include/mbedtls/cipher.h

```diff
@@ -953,7 +953,7 @@ int mbedtls_cipher_update( mbedtls_cipher_context_t *ctx,
  * \param ctx           The generic cipher context. This must be initialized and
  *                      bound to a key.
  * \param output        The buffer to write data to. This needs to be a writable
- *                      buffer of at least \p block_size Bytes.
+ *                      buffer of at least \c block_size Bytes.
```

Comment:
I think that instead of giving a specific example we should have a similar format of documentation as `mbedtls_cipher_update` has. There, `block_size` is mentioned. I removed the \c part to notify the reader that we're not talking about any specific variable.